### DISCUSSION TITLE
Websocket support

### DISCUSSION THREAD

**Discussion by marclave** [scalar/scalar#7797] [Ideas]

**Comment by Makeuptehrani** [comment]
Is it supported for Android?

**Comment by Its4Nik** [comment]
@hanspagel is there any plan on this being done in the near future? I am currently writing an API with websockets and would love this feature. 

If not, could you tell me how this should be integrated? Then I would try and see what I could do.

**Comment by MarkLee425** [comment]
> [@hanspagel](https://github.com/hanspagel) is there any plan on this being done in the near future? I am currently writing an API with websockets and would love this feature.
> 
> If not, could you tell me how this should be integrated? Then I would try and see what I could do.

Same here, I am currently using websocket with swagger and swagger-socket. Thinking to change this if there is support of websocket 🤔

**Comment by MrSquaare** [comment]
More generally, this could support of the [AsyncAPI specification](https://www.asyncapi.com/docs/reference/specification/v3.0.0)

**Comment by riprsa** [comment]
ngl, i would love to see WS support

**Comment by nathan-muon** [comment]
Would love to see this as well!

**Comment by hongfeiyang** [comment]
woudl liove to see this
